lr-cli/cortx_setup/commands/cluster/config/component.py

```diff
@@ -79,13 +90,23 @@ def run(self, component_group):
                         cortx_components[component_group],
                         stages=['config.config', 'config.init_mod']
                     )
-                self.logger.debug(f"Deploying ha components on nodes")
+                self.logger.debug(f"Deploying cortx ha components on {primary}")
                 self._configure(
                     ['ha.cortx-ha'],
                     stages=[
-                        'config.post_install','config.prepare',
-                        'config.config', 'config.init_mod']
+                        'config.post_install', 'config.prepare',
```

Comment:
**Below phases should be executed in parallel on all nodes:**
`'config.post_install', 'config.prepare',`